### DISCUSSION TITLE
GitHub Copilot login only works when starting IntelliJ as administrator

### DISCUSSION THREAD

**Discussion by dammg** [community/community#67936] [Copilot Conversations] [answered]
### Select Topic Area

Question

### Body

```
OS: Windows 11
IntelliJ version: 2023.2 (Ultimate Edition) Build #IU-232.8660.185
Copilot plugin version: 1.2.22.3129
Copilot Business subscription (access to Copilot managed by organisation)
```

Hello GitHub team!

I am trying to get to work the GitHub Copilot plugin for IntelliJ, but unfortunately I can only manage to log in, when starting IntelliJ as an administrator. We are using company issued Windows 11 notebooks, that have restricted priviliges by default. We do have access to a local admin account but it should only be used for installing third-party software, not for running our IDE.

When I try the login with my default user, I get the following message from Copilot:

`Failed to initiate the GitHub login process. Please try again.`

The IntelliJ logs produce the following lines:

```
FINE - #com.github.copilot.lang.agent.UnavailableAgentProcessService - agent service is unavailable: error in constructor, command: com.github.copilot.lang.agent.commands.SignInInitiateCommand@6e6e9aa1
FINE - #com.github.copilot.util.LoggerUtil - 
org.jetbrains.concurrency.MessageError: service is unavailable
```

We do not have a proxy, so it should not be a connection issue. Executing "curl --verbose https://copilot-proxy.githubusercontent.com/_ping" returns a 200 OK response.

Do you have any idea, what the underlying issue could be?

Thanks in advance!

Best regards,
Georg

**Comment by svabrad** [comment]
I am getting the exact same error. 

```
OS: Windows 11
IntelliJ version: IntelliJ IDEA 2023.2.2 (Ultimate Edition) Build #IU-232.9921.47
Copilot plugin version: 1.3.1.3453
Copilot Business subscription (access to Copilot managed by organisation)
```

We do use proxy and executing "curl -x http://proxy-to-internet:3128 -i -L https://copilot-proxy.githubusercontent.com/_ping" 
returns OK response.

**Comment by dammg** [comment, marked as answer]
We found the problem after all. It was caused by the Windows AppLocker, which is configured centrally by our IT. It blocked the execution of "copilot-agent-win.exe".

After whitelisting the plugin exe file, it now works without issues.

**Reply by svabrad** [reply]
This was the solution to our issue as well. Thank you!
